src/dbos-executor.ts

```diff
@@ -376,13 +360,9 @@ export class DBOSExecutor implements DBOSExecutorContext {
       const knexConfig: Knex.Config = {
         client: 'postgres',
         connection: {
-          host: userDBConfig.host,
-          port: userDBConfig.port,
-          user: userDBConfig.user,
-          password: userDBConfig.password,
-          database: userDBConfig.database,
+          connectionString: userDBConfig.connectionString,
           ssl: userDBConfig.ssl,
```

Comment:
Why are we still passing this in?  Shouldn't this information always be in the connection string?

Reply:
Let me check. I might have been misled that ssl was having the same limitation than connectTimeout and pool (i.e., knex/typeorm don't always respect what's in the connection string.)